### ISSUE TITLE
Holiday app lacks an H1

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/design-gc-conception#2088]
Note: This is an issue specific to the holiday app implementation of the side nav and (I think) doesn't concern the main component. Hence the low priority.

ITAO audit result: 

All pages in the solution lack a meaningful heading structure. More specifically:

1. Each page lacks an H1 heading.
2. Strings of text that visually appear as headings are not marked as headings.

**Impact on Users:**

Headings in communicate the organization of the content on the page. Web browsers, plug-ins, and assistive technologies can use them to provide in-page navigation.

Thus if there is a lack of a meaningful heading structure, users of assistive technologies will not receive the same understanding of the content on a page as other users.

**Sample Code (required):**

<h2 class="gcds-side-nav__heading">Canada holidays</h2>
**Recommendation (required if not SAMS/OCMC):**

1. Ensure that there are meaningful headings only on the page.

2. Ensure that they follow the correct order with H1 representing the main content and then H2s representing subheadings with H3s representing sub-subheadings and so on and so forth.

3. Ensure that every heading of the same level, appears in the same font size.

**Additional Resources:**

https://www.w3.org/WAI/tutorials/page-structure/headings/
**Steps to Reproduce:**

1. Open the solution: https://fable-test.d2fzvkr34ak02t.amplifyapp.com/

2. Locate all headings on the page.
 
<img width="1024" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1pkREE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4d697c839908f69885d594d6622b72c62a3252cd/image.png" alt="image.png" />
